### DISCUSSION TITLE
Why not consider the Knapsack problem as an environment？

### DISCUSSION THREAD

**Discussion by bxdgfw** [ai4co/rl4co#199] [General]
I think the Knapsack problem is quite different from the Routing Problems and Scheduling Problems already in the document, so it can have good reference value in the creation and use of some new problems. Thanks

**Comment by fedebotu** [comment]
Thanks for your suggestion. We could refer to POMO's example: https://github.com/yd-kwon/POMO/blob/master/OLD_ipynb_ver/POMO_KP/source/knapsack_problem.py
